src/test/java/com/zepben/evolve/services/common/translator/CimPbTest.kt

```diff
@@ -143,6 +143,17 @@ class CimPbTest {
     private fun testNetworkService(sourceNetworkService: NetworkService) {
         val targetNetwork = NetworkService()
 
+        // Handle EquipmentContainers, OperationalRestrictions, and ConnectivityNodes first, so that their equipment/terminal references
```

Comment:
Is this change required for the target network to remain valid? If so I believe the change is hiding a bug as added things in either order should resolve the references

Reply:
you're right - the bug is because the reverse resolver never gets executed. i'm having a look at how to fix.

Reply:
OK fixed by storing the reverse resolver with the UnresolvedReference and resolving it in `BaseService.add`. I couldn't think of any better way, so let me know what you think